src/Symfony/Component/AutoMapper/AutoMapperNormalizer.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AutoMapper;
```

Comment:
Shouldn't this class be moved in the Serializer component?

Reply:
Yes, it should, i added that here as a preview, but some classes preent in this component should be into other components

Reply:
@dunglas i don't think it's a good idea to add this normalizer into the serializer on this PR should i just remove it ATM ?